packages/synapse-react-client/src/components/AclEditor/ResourceAccessItem.tsx

```diff
@@ -8,22 +8,18 @@ import { IconSvgButton } from '../IconSvgButton'
 
 export const REMOVE_BUTTON_LABEL = 'Remove from AR Permissions'
 
-const availablePermissionLevels: PermissionLevel[] = [
```

Comment:
These are Access Requirement-specific permission levels, so move them to the AR ACL editor file. `availablePermissionLevels` is now passed as a prop so each editor can be configured for the resource it supports.